cmd/git-sync/main.go

```diff
@@ -659,7 +660,7 @@ func setupGitSSH(setupKnownHosts bool) error {
 	}
 
 	if setupKnownHosts {
-		_, err := os.Stat(pathToSSHKnownHosts)
+		_, err = os.Stat(pathToSSHKnownHosts)
```

Comment:
rationale: `err` assignment on 668 affects only `if` scope so error check on 674 won't see the error if it occurs.

Reply:
good one.